src/ui/windows/TogglDesktop/TogglDesktop/ui/Tutorial/BasicTutorialScreen7.xaml

```diff
@@ -5,24 +5,30 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:tutorial="clr-namespace:TogglDesktop.Tutorial"
              xmlns:toggl="clr-namespace:TogglDesktop"
+             xmlns:system="clr-namespace:System;assembly=mscorlib"
              mc:Ignorable="d" 
              d:DesignHeight="559.069" d:DesignWidth="381.927"
              Style="{StaticResource TutorialScreen}">
+    <UserControl.Resources>
+        <system:Double x:Key="TabsHeight">40</system:Double>
+    </UserControl.Resources>

```

Comment:
This constant could be moved into some *Constants.cs (or some xaml with constants) class. We have one more place where this number 40 is used - it's `TimeEntryListView,xaml` height or border (line 29). It would be nice for both heights to use the same constant.

Reply:
fixed